apps/question-bank/src/main/java/gov/cdc/nbs/questionbank/page/PageStateChanger.java

```diff
@@ -51,13 +58,20 @@ public PageStateResponse savePageAsDraft(Long id) {
             Optional<WaTemplate> result = templateRepository.findById(id);
             if (result.isPresent()) {
                 WaTemplate page = result.get();
+                if(page.getTemplateType().equals(PageConstants.PUBLISHED_WITH_DRAFT))  {
+                response.setMessage(PageConstants.SAVE_DRAFT_NOCHANGE);
+                response.setTemplateId(page.getId());	
+                }
```

Comment:
I think we would want to throw a  `BadRequest` exception of some kind here so that the status code returned by the API is non 200.  Maybe a `PageUpdateException`?

Reply:
Okay sure  I will update

Reply:
Hi @mpeels  updated.